pkg/api/latest/latest.go

```diff
@@ -117,9 +117,10 @@ func init() {
 	// the list of kinds that are scoped at the root of the api hierarchy
 	// if a kind is not enumerated here, it is assumed to have a namespace scope
 	kindToRootScope := map[string]bool{
-		"Node":      true,
-		"Minion":    true,
-		"Namespace": true,
+		"Node":             true,
+		"Minion":           true,
+		"Namespace":        true,
+		"PersistentVolume": true,
```

Comment:
I think we're going to want PersistentVolumes to be inside namespaces.  In the future general case, individual users will be able to create their own PV's, and want to put them in namespaces to avoid clashes.  In the first iteration, where the "storage administrator" creates all PV's on behalf of users, would there be any downside to having those inside namespaces too (even if it's just a global "admin-created-volumes" namespace or similar)?